--- requests-2.12.4/tests/testserver/server.py	2016-12-21 11:31:56.000000000 -0800
+++ requests-2.12.4/tests/testserver/server.py.new	2016-12-30 10:40:06.085995065 -0800
@@ -29,7 +29,7 @@ class Server(threading.Thread):
     def __init__(
         self,
         handler=None,
-        host="localhost",
+        host="127.0.0.1",
         port=0,
         requests_to_handle=1,
         wait_to_close_event=None,
